docker/actions/Dockerfile.actions.bionic

```diff
@@ -8,6 +8,8 @@ RUN apt-get -y update && \
       rm -rf /var/cache/apt/archives && \
       rm -rf /var/lib/apt/lists
 
+RUN python3 -m pip install --upgrade numpy scipy
+
```

Comment:
For NumPy, I believe this is due to the `default_rng` API which is not available on Bionic's shipped version of NumPy. For SciPy,  it actually should not be there anymore because of [536701c](https://github.com/FluidityProject/fluidity/pull/345/commits/536701cb9b0aeff5623c874b4b38f79d15f3b4cd) and [30f8c69](https://github.com/FluidityProject/fluidity/pull/345/commits/30f8c69123015f78a717065fcd0aa4e14f530f87). What is your recommendation here?

Reply:
I see. In general this wouldn't be a good idea: if we claim to support out of the box on Ubuntu X (with some documented additional stuff alongside) then that's what we should be testing. If we are depending on a newer version of numpy or scipy - which is perfectly reasonable - then we should we should document that somewhere, and give instructions on how to upgrade on Ubuntu X (and telling people to `pip install --upgrade` their numpy in their system python is not ideal) add it to the *base* image, and push that to Docker.

Since this is just Bionic, which we might drop soon anyway (probably when 24.04 comes out) I don't really care, let's just merge it.